includes/collection/class-interactions.php

```diff
@@ -111,14 +111,13 @@ public static function add_reaction( $activity ) {
 			return false;
 		}
 
-		$type = $activity['type'];
+		$comment_type = Comment::get_comment_type_by_activity_type( $activity['type'] );
```

Comment:
Could there ever be multiple comment types for an activity type? Is that something we have to worry about?

Reply:
I don't think so... there is the possibility that we might store items in different places (when we think about direct mentions for example), but that is a different discussion...